features/cart/shopping_cart/adding_product_with_variants_with_discounted_catalog_price_to_cart.feature

```diff
@@ -7,14 +7,14 @@ Feature: Adding a product with selected variant with discounted catalog price to
     Background:
         Given the store operates on a single channel in "United States"
         And the store has a "T-Shirt" configurable product
-        And this product has "PHP T-Shirt" variant priced at "$20"
-        And this product has "Kotlin T-Shirt" variant priced at "$400"
+        And the product "T-Shirt" has a "PHP T-Shirt" variant priced at "$20"
+        And the product "T-Shirt" has a "Kotlin T-Shirt" variant priced at "$400"
         And the store has a "Keyboard" configurable product
-        And this product has "RGB Keyboard" variant priced at "$40"
-        And this product has "Pink Keyboard" variant priced at "$300"
+        And the product "Keyboard" has a "RGB Keyboard" variant priced at "$40"
+        And the product "Keyboard" has a "Pink Keyboard" variant priced at "$40"
         And there is a catalog promotion "Winter sale" that reduces price by "25%" and applies on "PHP T-Shirt" variant
```

Comment:
why did you changed it? imo with shared storage it looks better

Reply:
I have problems with ApiContexts